### PR TITLE
[beta] Add terraform SCC and CPI information

### PR DESCRIPTION
**Pull request by mjura**

### CODE REVIEW

File: adoc/deployment-ecp.adoc [hunk 1]
```diff
@@ -63,17 +63,34 @@ You can set the timezone before deploying the nodes by modifying the following f
...
++
+. You can also enable Cloud Provider Integration with OpenStack in `~/my-cluster/cpi.auto.tfvars`:
++
+----
+# Enable CPI integration with OpenStack
+cpi_enable = true
+
+# Used to specify the path to your custom CA file
+# for example "/etc/ssl/certs/SUSE_Trust_Root.pem"
+#ca_file = ""
```
Comment (kravciak) [inline adoc/deployment-ecp.adoc]:
```suggestion
#ca_file = "/etc/ssl/certs/<CUSTOM_CA_FILE>"
```

File: adoc/deployment-ecp.adoc [hunk 2]
```diff
@@ -63,17 +63,34 @@ You can set the timezone before deploying the nodes by modifying the following f
...
 +
 This is required so all the deployed nodes can automatically register to {scc} and retrieve packages.
++
+. You can also enable Cloud Provider Integration with OpenStack in `~/my-cluster/cpi.auto.tfvars`:
++
+----
+# Enable CPI integration with OpenStack
+cpi_enable = true
+
+# Used to specify the path to your custom CA file
```
Comment (kravciak) [inline adoc/deployment-ecp.adoc]:
```suggestion
# Used to specify the name of your custom CA file located in /etc/ssl/certs.
# Upload CUSTOM_CA_FILE to this path on nodes before joining them to your cluster.
```

**Comment by nkoranova** [comment]
@mjura except for this being a PR to beta and not master and reviewed by Martin, does it differ in any way from https://github.com/SUSE/doc-caasp/pull/382?

File: adoc/deployment-ecp.adoc (hunk 1 above)
Comment (mjura) [reply adoc/deployment-ecp.adoc]: Fixed

File: adoc/deployment-ecp.adoc (hunk 2 above)
Comment (mjura) [reply adoc/deployment-ecp.adoc]: fixed

**Comment by mjura** [comment]
@nkoranova ok, I have closed #382, please merge this PR

**Pull request merged**